clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -9913,11 +9913,11 @@ def warn_new_dangling_initializer_list : Warning<
   "will be destroyed at the end of the full-expression">,
   InGroup<DanglingInitializerList>;
 def warn_unsupported_lifetime_extension : Warning<
-  "sorry, lifetime extension of "
+  "lifetime extension of "
   "%select{temporary|backing array of initializer list}0 created "
-  "by aggregate initialization using default member initializer "
+  "by aggregate initialization using a default member initializer "
   "is not supported; lifetime of %select{temporary|backing array}0 "
```

Comment:
Perhaps "is not yet supported" or similar. As noted at https://github.com/llvm/llvm-project/issues/61256#issuecomment-1477097827, the "sorry" kind of indicated that it was Clang's fault, rather than the code doing something wrong.